website/ref/builtin.md

```diff
@@ -4,8 +4,21 @@
 
 The builtin module contains facilities that are potentially useful to all users.
 It occupies the `builtin:` namespace. You rarely have to explicitly specify the
-namespace though, since it is one of the namespaces consulted when resolving
-unqualified names.
+namespace since it is one of the namespaces consulted when resolving unqualified
+names. The exception to this rule is if you define a function with the same name
+as a builtin. In which case to use the builtin rather than the function you need
```

Comment:
I'd always write "which" clauses as part of the same sentence but I'll fix that myself.